packages/graphql-validate-fixtures/src/index.ts

```diff
@@ -75,11 +188,24 @@ export async function evaluateFixtures(
       }
     }),
   );
-  const ast = compile(schema, document);
 
-  return runForEachFixture(fixturePaths, (fixture) =>
-    validateFixtureAgainstAST(fixture, ast),
-  );
+  let schema: GraphQLSchema;
+
+  try {
+    schema = projectConfig.getSchema();
+  } catch (error) {
+    throw new Error(
```

Comment:
If we get to this point, I would not try to merge it in with other "missing operation" errors — I'd rather just do a full stop and emit an error.

Reply:
I believe that is the case, these errors during the operation discovery phase should all be going up the stack. the validation error processing should only happen in the evaluation phase. I will double check to confirm this.

Reply:
[confirmed](https://github.com/Shopify/graphql-tools-web/pull/21/files#diff-1acdbb166626467a9e66c1c258aee06eR129). we do not handle these types of errors, they pop all the way out to the entrypoint.